src/entities/list/components/ListDetails.tsx

```diff
@@ -307,7 +309,11 @@ export const ListDetails = ({ admins, listId, listDetails, savedUsers }: ListDet
                 )}
               </button>
               <div className="font-semibold">{listDetails && listDetails?.upvotes?.length}</div>
-              <SocialsShare variant="icon" />
+              <SocialsShare
+                shareText={`support this List on ${PLATFORM_NAME}  by donating  to Projects or 
```

Comment:
```suggestion
                shareText={`support this List on ${PLATFORM_NAME} by donating to Projects or 
```

Redundant whitespaces